docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -205,10 +205,27 @@ An executable `fhir-persistence-schema` jar can be downloaded from the project's
 
 For a detailed guide on configuring IBM Db2 on Cloud for the IBM FHIR Server, see [DB2OnCloudSetup](https://ibm.github.io/FHIR/guides/DB2OnCloudSetup).
 
+##### 3.4.1.1.3 PostgreSQL
+If you configure the FHIR server to use a PostgreSQL database, you must:
+
+1. create the database if it doesn't already exist
+
+2. execute the `fhir-persistence-schema` utility with a db-type of `postgresql` to create the necessary schemas (tables, indices, stored procedures, etc)
+
+An executable `fhir-persistence-schema` jar can be downloaded from the project's [Releases tab](https://github.com/IBM/FHIR/releases) and documentation can be found at https://github.com/IBM/FHIR/tree/master/fhir-persistence-schema.
+
+To use the
+For a detailed guide on configuring IBM Db2 on Cloud for the IBM FHIR Server, see [DB2OnCloudSetup](https://ibm.github.io/FHIR/guides/DB2OnCloudSetup).
```

Comment:
good catch; I think I was going to write a bit about using IBM Cloud Databases for PostgreSQL but never got back to it.  Lets just remove this for now.
```suggestion
```